recipes/ditaa/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# if some command fails, abort immediately
+set -ex
+
+mkdir -vp ${PREFIX}/bin;
```

Comment:
No need for semilcolons at the end of line

Reply:
Updated in the recent commit